### PR TITLE
Install dropbox

### PR DESCRIPTION
**Pull request by jburel**
install dropbox from pypi

### CODE REVIEW

File: omero/sysadmins/dropbox.rst [hunk 1]
```diff
@@ -37,6 +37,29 @@ the following more specific requirements:
     operating system. Watching a network-attached share (NAS) is strictly
     ***not*** supported.
 
+Installing DropBox
+------------------
+
+From OMERO 5.6.0 release, the library ``omero-dropbox`` supports Python 3 and
```
Comment (mtbc) [inline omero/sysadmins/dropbox.rst]:
```suggestion
From the OMERO 5.6.0 release, the library ``omero-dropbox`` supports Python 3 and
```

**Comment by dominikl** [comment]
Trying to follow the doc... first thing I asked myself is, why do I have to create an new venv? And for which user? I guess must be as root using the `/opt/omero/server/venv3` so I installed it with `/opt/omero/server/venv3/bin/pip install omero-dropbox==5.6.dev3` (Typo in the doc there, there's no dev10, only dev3).

Enabled dropbox via (as omero-server user):
```
$ omero admin ice server enable MonitorServer
$ omero admin ice server enable DropBox
```
This should actually be part of the section "Installing DropBox", not only "Permissions".

 Then got a lot of errors like this in the Dropbox.log (testing on the debian10 VM again):
```
2020-01-13 12:47:50,208 INFO  [                        fsclient.DropBox] (MainThread) Monitor parameters = {'default': {'watchDir': '', 'eventTypes': [Creation, Modification], 'pathMode': Follow, 'whitelist': [], 'blacklist': [], 'timeout': 0.0, 'blockSize': 0, 'ignoreSysFiles': True, 'ignoreDirEvents': True, 'dirImportWait': 60, 'fileWait': 15.0, 'throttleImport': 10, 'timeToLive': 0, 'timeToIdle': 600000, 'fileBatch': 10, 'readers': '', 'importArgs': ''}}
2020-01-13 12:47:50,220 ERROR [                        fsclient.DropBox] (MainThread) Failed to use a query service :
Traceback (most recent call last):
  File "/opt/omero/server/OMERO.server/lib/python/fsDropBox.py", line 127, in run
    watchDir = pathModule.path(
AttributeError: type object 'path' has no attribute 'path'
```
Uninstalled it again. Wanted to try conda, but same question here, don't know which user and where am I supposed to create this new venv?

**Comment by dominikl** [comment]
Thanks, much clearer now 👍

**Comment by jburel** [comment]
Merging

**Pull request merged**